distribution/pom.xml

```diff
@@ -0,0 +1,125 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
It looks like IDEA generates these tilde signs for XML files after I configured the copyright files.  Will remove them.